### PR TITLE
upgraded cedar-go to v0.3.2

### PR DESCRIPTION
**Pull request by mqf20**
Upgrading the `cedar-go` version so that https://github.com/cedar-policy/cedar-go/issues/30 can proceed

### CODE REVIEW

**Comment by mqf20** [comment]
@khieta could you have a look please?

File: tinytodo-go/README.md
Comment (khieta) [inline tinytodo-go/README.md:27]: Can you add a note somewhere in the README that says that this example uses v0.3.2?

File: tinytodo-go/internal/app/server/entitystore/action/action.go
Comment (khieta) [inline tinytodo-go/internal/app/server/entitystore/action/action.go:11]:
```suggestion
// Action is an enum that represents the different entity types supported by TinyTodo.
```

File: tinytodo-go/internal/app/server/entitystore/entityuid/entityuid.go
Comment (khieta) [inline tinytodo-go/internal/app/server/entitystore/entityuid/entityuid.go:37]: This function name could probably just be `Parse`, given that it's already in `entityuid` package.

**Review by khieta — APPROVED** [review]
Thanks for the contribution! I'll ping the team to get a second review se we can get this merged.

File: tinytodo-go/internal/app/server/entitystore/entityuid/entityuid.go
Comment (mqf20) [reply tinytodo-go/internal/app/server/entitystore/entityuid/entityuid.go]: fixed in 003a5f4633414efc9bf8a4abc27ebe4737834286

Comment (mqf20) [reply tinytodo-go/internal/app/server/entitystore/entityuid/entityuid.go]: On second thought, should `ParseEntityUID` be named `ParseCedarLanguageRepresentation` (or `ParseCedarRepresentation`) instead? This would be more explicit.

Comment (mqf20) [reply tinytodo-go/internal/app/server/entitystore/entityuid/entityuid.go]: Or perhaps `NewFromCedarLanguageRepresentation` (or `NewFromCedarRepresentation`)

File: tinytodo-go/README.md
Comment (mqf20) [reply tinytodo-go/README.md]: addressed in 7e631e66ea59aa9b7ee84450191eb17b18c4b4cd - please see if the note is acceptable

**Review by aaronjeline — APPROVED** [review]

**Review by cdisselkoen — APPROVED** [review]
Happy to approve this based on @khieta's and @aaronjeline's reviews.

**Pull request merged**
